src/sorcha/modules/PPAddUncertainties.py

```diff
@@ -191,7 +196,10 @@ def uncertainties(
 
     if configs.get("trailing_losses_on", False):
         dMag = PPTrailingLoss.calcTrailingLoss(
-            detDF[dra_name] * degCos(detDF[dec_name]), detDF[ddec_name], detDF[seeingName]
```

Comment:
On other change:

AstRARate(deg/day) is RARateCosDec_deg_day in the new ephemeris generation so you shouldn't be multiplying be cosdec here 

https://github.com/dirac-institute/sorcha/blob/3279da4cd19954917f445b0b3639d53ac3f431c1/src/sorcha/modules/PPTrailingLoss.py#L88C1-L89C48